org.sugarlabs.PyEyes.json

```
@@ -0,0 +1,60 @@
+{
+    "app-id": "org.sugarlabs.PyEyes",
+    "base": "org.sugarlabs.BaseApp",
+    "base-version": "23.06",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "separate-locales": false,
+    "command": "sugarapp",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+        "--env=SUGAR_BUNDLE_ID=org.sugarlabs.PyEyes",
+        "--env=SUGAR_BUNDLE_PATH=/app/share/sugar/activities/PyEyes.activity"
+    ],
+    "modules": [
+        {
+            "name": "pyeyes",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sugarlabs/pyeyes-activity.git",
+		    "commit": "478fcaf51df5556a3fc26503dfbb63d9d13d1405"
+                },
```

The indentation is wrong here
```suggestion
                    "url": "https://github.com/sugarlabs/pyeyes-activity.git",
		                "commit": "478fcaf51df5556a3fc26503dfbb63d9d13d1405"
                },
```

The indentation is wrong here